spec/sidekiq/evss/disability_compensation_form/submit_form0781_spec.rb

```diff
@@ -171,6 +171,44 @@
           end
         end
       end
+
+      context 'when the API Provider uploads are enabled' do
+        before do
+          Flipper.enable(:disability_compensation_use_api_provider_for_0781_uploads)
```

Comment:
Enabling and disabling a feature flag in tests can lead to some intermittent test failures. Can you mock it instead? 
```ruby
allow(Flipper).to receive(:enabled?).with(:disability_compensation_use_api_provider_for_0781_uploads).and_return(true)
```
We recently updated this example to show that: https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample

Reply:
@rmtolmach Thanks Rebecca! Good call. I updated the three flippers in this file to use mocks instead. 

Excellent documentation, btw.